src/main/webapp/i18n/de/exam.json

```
@@ -312,9 +312,12 @@
             "pendingChanges": "Alle ungespeicherten Änderungen gehen beim Verlassen der Klausur verloren. Bist du sicher, dass du fortfahren möchtest?",
             "handInEarly": "Vorzeitig abgeben",
             "continueAfterHandInEarly": "Weitermachen",
+            "attendanceCheck": "Anwesenheitskontrolle",
             "continueAfterHandInEarlyDescription": "Du willst doch nicht vorzeitig abgeben?",
             "handInEarlyNoticeFirstSentence": "Du bist dabei die Klausur vorzeitig abzugeben.",
             "handInEarlyNoticeSecondSentence": "Du kannst danach nicht mehr an der Klausur teilnehmen.",
+            "attendanceNotVerifiedWarningFirstSentence": "Deine Anwesenheit wurde noch nicht kontrolliert.",
+            "attendanceNotVerifiedWarningSecondSentence": "Bitte kontaktiere die Hörsaalaufsicht nach der Prüfung, um deine Anwesenheit zu kontrollieren.",
```

Why "after the exam"?
```suggestion
            "attendanceNotVerifiedWarningSecondSentence": "Bitte kontaktiere die Hörsaalaufsicht, um deine Anwesenheit zu kontrollieren.",
```

---

My idea was that in case a student wants to hand-in early, they can, but they shouldn't contact the instructors at this very moment so that instructors don't have to change the order of attendance checks. Either we can remove "after the exam" or change it to something like "wait for your attendance to be checked or contact the instructors after the exam has ended". I'm open to your suggestions.